data/human/culture free worlds.txt

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+mission "ZZCU South: The Great Pig War"
+	landing
+	to offer
+		and
+			month == 6
+			day >= 10
+			day <= 20
+	source
+			attributes "spaceport" "farming" "south"
+	on offer
+		conversation
+			`Farm world spaceports tend to be lonely affairs outside of harvesting season, but this one is bustling with festive energy. Rows of colorful tents and throngs of the planet's natives fill the area, as well as a proliferation of craft-work pig-sculptures; some are built from hay, some papier-mache and paint - one even appears to have been carved from butter. Most spaces that aren't taken by a silo, barn, or hangar of some sort have had a refreshment stand, game, or exhibition set up - the remaining few patches of ground are occupied by the feet of the planet's inhabitants. Live music fills the air, cheap beer flows like water, and fried food sizzles enticingly as it vacates vats of oil.`
+			choice
+				`	(Carry on with your business. You can always come back next year.)`
```

Comment:
"Action" choices (those in parentheses) are always written in first person perspective.
Important to note though that we don't actually know that this is a yearly festival.
```suggestion
				`	(Carry on with your business. I can always come back next year.)`
```